### PR TITLE
Move mentions loading text out of header

### PR DESCRIPTION
**Pull request by MaxLeiter**
Makes it less obvious when the text updates

### CODE REVIEW

File: client/components/Mentions.vue [hunk 1]
```diff
@@ -8,10 +8,10 @@
 		<div class="mentions-popup">
 			<div class="mentions-popup-title">
 				Recent mentions
-				<span v-if="isLoading">- Loading…</span>
 			</div>
 			<template v-if="resolvedMessages.length === 0">
-				<p>There are no recent mentions.</p>
+				<span v-if="isLoading">Loading…</span>
+				<p vu-if="!isLoading">There are no recent mentions.</p>
```
Comment (xPaw) [inline client/components/Mentions.vue:14]: Typo.
```suggestion
				<p v-else>There are no recent mentions.</p>
```

Comment (richrd) [reply client/components/Mentions.vue:14]: I'm surprised this isn't caught by tests & lint :thinking:

Comment (xPaw) [reply client/components/Mentions.vue:14]: Well it's just an attribute, how will the linter know?

Comment (richrd) [reply client/components/Mentions.vue:14]: I thought it might check all valid HTML5 and Vue attributes as well as `data-*` but I guess not.

Comment (MaxLeiter) [reply client/components/Mentions.vue]: Fixed

Comment (xPaw) [reply client/components/Mentions.vue]: Can you squash please

File: client/components/Mentions.vue [hunk 2]
```diff
@@ -8,10 +8,10 @@
 		<div class="mentions-popup">
 			<div class="mentions-popup-title">
 				Recent mentions
-				<span v-if="isLoading">- Loading…</span>
 			</div>
 			<template v-if="resolvedMessages.length === 0">
-				<p>There are no recent mentions.</p>
+				<span v-if="isLoading">Loading…</span>
```
Comment (xPaw) [inline client/components/Mentions.vue:13]: Oh maybe this should `<p>` tag too.

**Pull request merged**